WCF/Version.props

```diff
@@ -6,14 +6,14 @@
       Update for every public release. 
     -->
     <SemanticVersionMajor>0</SemanticVersionMajor>
-    <SemanticVersionMinor>25</SemanticVersionMinor>
+    <SemanticVersionMinor>26</SemanticVersionMinor>
     <SemanticVersionPatch>0</SemanticVersionPatch>
 
     <!-- 
       Date when Semantic Version was changed. 
       Update for every public release.
     -->
-    <SemanticVersionDate>2016-03-17</SemanticVersionDate>
+    <SemanticVersionDate>2016-06-21</SemanticVersionDate>
```

Comment:
@abaranch , do you remember whether we need to rev semverdate for the minor release? I thought it will reset file version and new file will have lower version than the old one.